src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
-        None, description="Number of GPUs per node.", title="Num Gpus Per Node"
+        None, description='Number of GPUs per node.', title='Num Gpus Per Node'
     )
     exclusive: Optional[bool] = Field(
-        None, description="To not share nodes with other running jobs.", title="Exclusive"
+        None, description='To not share nodes with other running jobs.', title='Exclusive'
     )
-    queue: Optional[str] = Field(None, description="Scheduler's queue.", title="Queue")
+    queue: Optional[str] = Field(None, description="Scheduler's queue.", title='Queue')
 
 
 class KubernetesKedaBackend(BaseModel):
-    plugin_name: Literal["kubernetes"] = Field(..., title="Plugin Name")
+    plugin_name: Literal['kubernetes'] = Field(..., title='Plugin Name')
     debug: Optional[bool] = Field(
-        False, description="Enable additional debugging of the backend", title="Debug"
+        False, description='Enable additional debugging of the backend', title='Debug'
     )
     job_script_template_path: Optional[str] = Field(
         None,
-        description="Path to the job script template to use in the backend.",
-        title="Job Script Template Path",
+        description='Path to the job script template to use in the backend.',
+        title='Job Script Template Path',
     )
-    working_dir: Optional[str] = Field(
-        None, description="Working directory to use in the backend.", title="Working Dir"
+    shared_dir: Optional[str] = Field(
+        None, description='Working directory to use in the backend.', title='Shared Dir'
     )
     env: Optional[Dict[str, Any]] = Field(
-        None, description="Static environment variables needed for job execution.", title="Env"
+        None, description='Static environment variables needed for job execution.', title='Env'
     )
     cpu_limit: Optional[str] = Field(
-        "1.0", description="CPU limit applied to each evaluator instance.", title="Cpu Limit"
+        '1.0', description='CPU limit applied to each evaluator instance.', title='Cpu Limit'
     )
     memory_limit: Optional[str] = Field(
-        "250M", description="Memory limit applied to each evaluator instance.", title="Memory Limit"
+        '250M', description='Memory limit applied to each evaluator instance.', title='Memory Limit'
     )
     namespace: Optional[str] = Field(
-        "default", description="Kubernetes namespace to use to scale evaluators.", title="Namespace"
+        'default', description='Kubernetes namespace to use to scale evaluators.', title='Namespace'
     )
     target_resource_kind: Optional[str] = Field(
-        "job",
-        description="Kubernetes resource kind that the REP scaler should target. Options are ``job``, ``deployment`` and ``statefulset``.",
-        title="Target Resource Kind",
+        'job',
+        description='Kubernetes resource kind that the REP scaler should target. Options are ``job``, ``deployment`` and ``statefulset``.',
```

Comment:
```suggestion
        description='Kubernetes resource kind that the REP scaler should target. Options are ``job``, ``deployment``, and ``statefulset``.',
```
We recommend alphabetizing the options, unless there is a reason not to,